### ISSUE TITLE
Animation don't work

### ISSUE THREAD

**Issue by devandrefernandes** [rive-app/rive-flutter#57]
Hello!! I'm a beginner in rive...

My rive animation displays first frame, but animation doesn't start.

```
rootBundle.load('assets/loading.riv').then((data) async {
        final file = RiveFile();
        if (file.import(data)) {
          final artboard = file.mainArtboard;
          _controller = SimpleAnimation('idle');
          _controller.isActive = true;
          artboard.addController(_controller);
          setState(() => _riveArtboard = artboard);
        }
      },
);
```
```
Container(
        width: 140,
        height: 140,
        child: _riveArtboard == null
            ? SizedBox()
            : Rive(artboard: _riveArtboard),
);
```

Can anybody help me? Thanks.
The riv file is attached.

**Comment by devandrefernandes** [comment]
Resolved.

I changed the name of the animation inside the rive editor and inserted the same name in SimpleAnimation.

It worked for me:

`_controller = SimpleAnimation('loading');`

![Captura de tela de 2021-02-14 13-46-03](https://user-images.githubusercontent.com/7664151/107883035-20a47f80-6ecb-11eb-9bb8-cd835bcc0feb.png)

I downloaded it again, imported it into the flutter and it worked perfectly

Thanks!
